include/psa/crypto_values.h

```diff
@@ -892,6 +892,34 @@
 #define PSA_MAC_TRUNCATED_LENGTH(mac_alg)                               \
     (((mac_alg) & PSA_ALG_MAC_TRUNCATION_MASK) >> PSA_MAC_TRUNCATION_OFFSET)
 
+/* In the encoding of a MAC algorithm, the bit corresponding to
+ * PSA_ALG_MAC_MINIMUM_LENGTH_FLAG encodes the fact that the algorithm is
+ * a usage algorithm, which allows any algorithm corresponding to the same
+ * base class and a tag length greater or equal than the one encoded in
```

Comment:
MAC algorithms generate a 'MAC' and not a 'tag'. The existing APIs and documentation are consistent about this. Can we maintain this consistency in the new APIs and documentation?

Reply:
Probably a copypaste oversight. Will fix.

Reply:
Fixed.